content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -614,12 +656,10 @@ jwt:
   {{% /tab %}}
   {{< /tabs >}}
 
-Importantly, the API server is not an OAuth2 client, rather it can only be
-configured to trust a single issuer. This allows the use of public providers,
-such as Google, without trusting credentials issued to third parties. Admins who
-wish to utilize multiple OAuth clients should explore providers which support the
-`azp` (authorized party) claim, a mechanism for allowing one client to issue
-tokens on behalf of another.
+##### Limitations
```

Comment:
```suggestion
###### Limitations
```

Reply:
That's a very deep heading; it suggests we could improve the way we lay things out.